rdbms/src/main/java/org/spine3/server/storage/jdbc/aggregate/query/CreateEventCountTableQuery.java

```diff
@@ -32,7 +32,12 @@
 import static org.spine3.server.storage.jdbc.aggregate.query.Constants.EVENT_COUNT_COL;
 import static org.spine3.server.storage.jdbc.aggregate.query.Constants.ID_COL;
 
-public class CreateEventCountQuery<I> extends Query {
+/**
+ * Query that creates new aggregate table with event counts if it does not exist.
```

Comment:
Does it creates a table and then populates with event counts? It's better to say `for aggregate event counts after the last snapshot`. We should be precise here.